gen-dirs.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -e
+cd templates
+FILES=$(find -follow)
+LATEST="9.6"
+
+pre_gen_cleanup() {
+    if [[ ${VERSIONS} == *${LATEST}* ]]; then
+        rm -rf ../latest
+    fi
+    for version in ${VERSIONS}; do
+        rm -rf ../${version}
+    done
+}
+
+# postgresql specific
+pg_generate_symlinks() {
+    echo "Creating symlinks:"
+    for version in ${VERSIONS}; do
+        cd ../${version}
+        echo "    ${version}/README.md -> ${version}/root/usr/share/container-scripts/postgresql/README.md"
+        ln -s root/usr/share/container-scripts/postgresql/README.md README.md
+        echo "    ${version}/test test"
+        ln -s ../test test
+    done
+    cd ..
+    if [[ ${VERSIONS} == *${LATEST}* ]]; then
+        echo "Moving ${LATEST} version to latest"
+        if [ -d "./latest" ]; then
+            echo "Failed to copy directory: ./latest already exists"
+            exit 1
+        fi
+        mv ${LATEST} ./latest
+        echo "Creating symlink ${LATEST}->latest"
+        ln -s latest ${LATEST}
+    fi
+}
+
+generate_file() {
+    if [ -L $1 ]; then
+        echo "Skipping symbolic link: $1"
+        return
+    elif [[ -d $1 && ! -d ../${version}/$1 ]]; then
+        mkdir ../${version}/$1
+    else
+        echo "Generating: ${version}/$1"
+        /usr/bin/dg --max-passes 1 --multispec ../specs/multispec.yml \
```

Comment:
Once this is moved to Makefile (common.mk probably), please ensure that instead of /usr/bin/dg you are using `$(DG)` variable; to allow anybody to replace the distgen version from local installation (virtualenv).